BUILD.md

```diff
@@ -264,13 +264,30 @@ The web server will be running on `localhost:8084`.
 
 #### Webpack dev server
 
-To develop with a webpack dev server, run:
+To develop with a webpack dev server:
 
-```bash
-bin/web dev
-```
+1. Install Linkerd on a Kubernetes cluster.
```

Comment:
Shouldn't this item go above instead, under the Web heading, as a prerequisite to both web standalone and webpack dev server?